opencti-platform/opencti-graphql/tests/01-unit/utils/syntax-test.ts

```diff
@@ -12,6 +12,8 @@ describe('Regex Pattern Tests', () => {
     expect('example.com').toMatch(domainChecker);
     expect('sub.example.co.uk').toMatch(domainChecker);
     expect('løveskateboards.com').toMatch(domainChecker);
+    expect('test._mysubdomain.mydomain.com').toMatch(domainChecker);
+    expect('test_mysubdomain.domain.io').toMatch(domainChecker);
     expect('invalid_domain.12_3').not.toMatch(domainChecker);
```

Comment:
maybe add a test that it's not working if there is a dot in the main name and no dots (exemple: invalid_domain)